bower.json

```diff
@@ -25,6 +25,6 @@
   "dependencies": {
     "medium-editor": "^5.0.0",
     "to-markdown": "^2.0.1",
-    "uglify-js": "^2.5.0"
+    "UglifyJS2": "https://github.com/mishoo/UglifyJS2.git#^2.5.0"
```

Comment:
@annayafi Sounds good! :+1: The build process is a bit strange in this repository–probably we'll change in the future. I don't really like how I did that templating stuff and function stringifying etc. But it works. :grin: